src/Symfony/Component/DomCrawler/Crawler.php

```diff
@@ -1215,7 +1213,7 @@ private function findNamespacePrefixes(string $xpath): array
      */
     private function createSubCrawler($nodes)
     {
-        $crawler = new static($nodes, $this->uri, $this->baseHref);
+        $crawler = new static($nodes, $this->uri, $this->baseHref, $this->html5Parser);
```

Comment:
We don't want to expose the html5Parser argument in the public API, just to have it for the creation of a subcrawler. Assign it below instead, as done for the document and namespaces.